src/encoded/schemas/matrix_file.json

```diff
@@ -215,7 +223,16 @@
                             "genes/cell",
                             "genes/nuclei",
                             "umi/cell",
-                            "umi/nuclei"
+                            "umi/nuclei",
+                            "cells/gene",
+                            "nuclei/gene",
+                            "percent mitochondrial/cell",
+                            "percent mitochondrial/nuclei",
```

Comment:
These are actually "percent UMI mitochondrial", since reads are collapsed into UMI for scRNAseq. On a side note, "percent reads in peaks" is a term taken from Signac. It could be considered inconsistent with matrix value unit of "fragment ends".

Reply:
Maybe we should go for consistency by changing that to 'percent in peaks/nuclei'?

Reply:
Yes, I like that better. I'll leave "percent mitochondrial" and change "percent reads in peaks".